controllers/nodefeaturediscovery_controller.go

```diff
@@ -157,49 +157,49 @@ func (r *NodeFeatureDiscoveryReconciler) Reconcile(ctx context.Context, req ctrl
 	}
 
 	// Check the status of the NFD Operator Worker ServiceAccount
-	if rstatus, err := r.getWorkerServiceAccountConditions(ctx); err != nil {
+	if rstatus, err := r.getWorkerServiceAccountConditions(ctx, instance); err != nil {
```

Comment:
Hmm, this PR seems to bring other changes, too. Like supporting `instance` config parameter(?)

Reply:
https://github.com/ArangoGutierrez/node-feature-discovery-operator/blob/devel/v1spec_ns/controllers/nodefeaturediscovery_status.go#L497

Reply:
before it was empty

Reply:
Haha, sorry I got totally confused by the `instance` in this PR 🙈